src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -139,6 +153,127 @@ inline bool is_one_dim_shapes(const ov::Shape& in_dims, const ov::Shape& out_dim
+    auto eltwise = std::dynamic_pointer_cast<ngraph::op::Eltwise>(node.get_node_shared_ptr());
+    if (!eltwise) return false;
+    return eltwise->eltwise_type == ELTWISE_TYPE::Prod;
+}
+
+static bool is_eltwise_add(const ngraph::Output<ngraph::Node>& node) {
+    auto eltwise = std::dynamic_pointer_cast<ngraph::op::Eltwise>(node.get_node_shared_ptr());
+    if (!eltwise) return false;
+    return eltwise->eltwise_type == ELTWISE_TYPE::Sum;
+}
+
+static bool is_pooling(const ngraph::Output<ngraph::Node>& node) {
+    return (std::dynamic_pointer_cast<ngraph::opset7::MaxPool>(node.get_node_shared_ptr()) != nullptr);
+}
+
+template <typename T>
+static bool is_Tbit_fq(const std::shared_ptr<ngraph::Node>& node) {
+    auto fq_node = std::dynamic_pointer_cast<ngraph::opset9::FakeQuantize>(node);
+    if (!fq_node)
+        return false;
+    auto levels = fq_node->get_levels();
+    return std::numeric_limits<T>::max() == levels;
+}
+
+static bool is_32bit_fq(const std::shared_ptr<ngraph::Node>& node) {
+    return is_Tbit_fq<uint32_t>(node);
+}
+
+static bool is_16bit_fq(const std::shared_ptr<ngraph::Node>& node) {
+    return is_Tbit_fq<uint16_t>(node);
+}
+
+static bool is_8bit_fq(const std::shared_ptr<ngraph::Node>& node) {
+    return is_Tbit_fq<uint8_t>(node);
+}
+
+static bool is_activation(const ov::Node* node) noexcept {
+    return ((dynamic_cast<const ngraph::opset9::Clamp*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Sigmoid*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Relu*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::op::ReLUIE*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Tanh*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::PRelu*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Exp*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Log*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Sign*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::Abs*>(node) != nullptr) ||
+            (dynamic_cast<const ngraph::opset9::SoftSign*>(node) != nullptr) ||
+            is_power_activation(node) ||
+            (dynamic_cast<const ngraph::opset9::FakeQuantize*>(node) != nullptr) ||
+            (dynamic_cast<const ov::intel_gna::op::Pwl*>(node) != nullptr) ||
+            (dynamic_cast<const ov::intel_gna::op::Identity*>(node) != nullptr));
+}
+
+static bool is_activation(const std::shared_ptr<ngraph::Node>& node) noexcept {
+    return is_activation(node.get());
+}
+
+static bool is_gna_precision_agnostic(std::shared_ptr<ngraph::Node> node) {
+    return ((std::dynamic_pointer_cast<ngraph::opset9::VariadicSplit>(node) != nullptr) ||
+            (std::dynamic_pointer_cast<ngraph::opset9::Split>(node) != nullptr) ||
+            (std::dynamic_pointer_cast<ngraph::opset9::Slice>(node) != nullptr) ||
```

Comment:
@sirzabek As I said previously these graph passes can't cover unaligned split and concats, because they are still handed in legacy code. That's why legacy call is kept. But when they will be moved to ngraph, they will be located before insertion of Identity. Also, please note that Identity is needed for plugin quantization only, for POT based models all precision changes are covered by FQs.

Reply:
@sirzabek Also I have added transformation test for split, it was uncovered.